ginga/canvas/types/basic.py

```diff
@@ -158,7 +159,7 @@ def contains_arr(self, x_arr, y_arr):
         # See: http://alienryderflex.com/polygon/
         xa, ya = x_arr, y_arr
 
-        result = numpy.empty(ya.shape, dtype=numpy.bool)
+        result = numpy.empty((ya.size, xa.size), dtype=numpy.bool)
```

Comment:
If you do not want to change `PolygonMixin.contains_arr()`, then the fix will either have to be in `BaseImage.get_shape_view()`, or even in `PolygonMixin.get_llur()`?